SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -320,6 +324,9 @@ void ASpatialDebugger::OnEntityAdded(const Worker_EntityId EntityId)
 			if (GetNetMode() == NM_Client)
 			{
 				LocalPlayerController->InputComponent->BindKey(ConfigUIToggleKey, IE_Pressed, this, &ASpatialDebugger::OnToggleConfigUI);
+				LocalPlayerController->InputComponent->BindKey(EKeys::RightMouseButton, IE_Pressed, this, &ASpatialDebugger::OnMousePress);
```

Comment:
Can we make these configurable similar to `ConfigUIToggleKey` ?

Reply:
Sure made these configurable.